### PR TITLE
Unfucks enthralling

### LINKED ISSUE #5055: (WEB REPORT BY: Anonymous REMOTE: ) Entrall Broken
>Revision (Should be above if you're viewing this from ingame!)
862ff2aac25c0822a7e22e64187919b889da022a

>General description of the issue
entrall is not working

>What you expected to happen
Being able to entrall people to enslave them

>What actually happened
When entralling someone I lose no blood and the other player gets a message of somekind but nothing more

>Steps to reproduce if possible
get blood, entrall someone

### PR DESCRIPTION
**Pull request by Intigracy**
replaces a bunch of spawns with sleeps

most of this code scares me deeply

fixes #5055
fixes partially #3506

### CODE REVIEW

File: code/game/gamemodes/vampire/vampire_powers.dm [hunk 1]
```diff
@@ -298,17 +301,17 @@
...
-		M.current << "<span class='warning'>You can only enthrall humans.</span>"
+		M.current << "<span class='warning'>You can only enthrall humanoids.</span>"
 		return
 
 	if(M.current.can_enthrall(C) && do_mob(M.current, C, (VAMP_CHARISMA in M.vampire.powers) ? 25 : 50)) //takes half the time with Charisma unlocked
 		if(!M.current.can_enthrall(C))
 			M.current << "<span class='warning'>Either you or your target moved, and you couldn't finish enthralling them!</span>"
 			return
-		if(!M.current.vampire_power(300, 0)) // recheck
+		if(M.current.vampire_power(300, 0)) // recheck
```
Comment (Intigracy) [inline code/game/gamemodes/vampire/vampire_powers.dm]: Here was the problem, for those curious.

Before it said:
If you don't have the blood necessary, then make the thrall. (doesn't work even without the blood due to var/mob/living/carbon/C = M.current.vampire_active(300, 0, 1) )

Now it says:
If you do have the blood neccessary, then make the thrall.

File: code/game/gamemodes/vampire/vampire_powers.dm [hunk 2]
```diff
@@ -95,7 +95,7 @@
 					M.current.adjustFireLoss(-2)
 					sleep(35)
 		M.current.verbs -= /client/proc/vampire_rejuvinate
-		spawn(200)
+		sleep(200)
 			M.current.verbs += /client/proc/vampire_rejuvinate
```
Comment (clusterfack) [inline code/game/gamemodes/vampire/vampire_powers.dm]: can you unindent this

File: code/game/gamemodes/vampire/vampire_powers.dm [hunk 3]
```diff
@@ -472,7 +476,7 @@
 			qdel(animation)
 			qdel(holder)
 		M.current.verbs -= /client/proc/vampire_jaunt
-		spawn(600) M.current.verbs += /client/proc/vampire_jaunt
+		sleep(600) M.current.verbs += /client/proc/vampire_jaunt
```
Comment (clusterfack) [inline code/game/gamemodes/vampire/vampire_powers.dm]: Also put the rest of this after sleep on the next line

File: code/game/gamemodes/vampire/vampire_powers.dm [hunk 4]
```diff
@@ -523,11 +527,11 @@
 			animation.layer = 5
 			//animation.master = src
 			usr.loc = picked
-			spawn(10)
+			sleep(10)
```
Comment (clusterfack) [inline code/game/gamemodes/vampire/vampire_powers.dm]: Keep this as a spawn

**Comment by clusterfack** [comment]
Add a changelog for this as well, format is as follows

filename - Intigracy_5174.yml
author: Intigracy
delete-after: true
changes:
- bugfix:
- rscadd:
- tweak:

**Comment by Intigracy** [comment]
Done.

**Pull request merged**